internal/provider/provider_sdk2.go

```diff
@@ -142,36 +164,36 @@ func getDataSourcesMap() map[string]*schema.Resource {
-		"mongodbatlas_online_archive":                                               mongodbatlas.DataSourceOnlineArchive(),
-		"mongodbatlas_online_archives":                                              mongodbatlas.DataSourceOnlineArchives(),
-		"mongodbatlas_ldap_configuration":                                           mongodbatlas.DataSourceLDAPConfiguration(),
-		"mongodbatlas_ldap_verify":                                                  mongodbatlas.DataSourceLDAPVerify(),
-		"mongodbatlas_search_index":                                                 mongodbatlas.DataSourceSearchIndex(),
-		"mongodbatlas_search_indexes":                                               mongodbatlas.DataSourceSearchIndexes(),
-		"mongodbatlas_data_lake_pipeline_run":                                       mongodbatlas.DataSourceDataLakePipelineRun(),
-		"mongodbatlas_data_lake_pipeline_runs":                                      mongodbatlas.DataSourceDataLakePipelineRuns(),
-		"mongodbatlas_data_lake_pipeline":                                           mongodbatlas.DataSourceDataLakePipeline(),
-		"mongodbatlas_data_lake_pipelines":                                          mongodbatlas.DataSourceDataLakePipelines(),
-		"mongodbatlas_event_trigger":                                                mongodbatlas.DataSourceEventTrigger(),
-		"mongodbatlas_event_triggers":                                               mongodbatlas.DataSourceEventTriggers(),
-		"mongodbatlas_project_invitation":                                           mongodbatlas.DataSourceProjectInvitation(),
-		"mongodbatlas_org_invitation":                                               mongodbatlas.DataSourceOrgInvitation(),
-		"mongodbatlas_organization":                                                 mongodbatlas.DataSourceOrganization(),
-		"mongodbatlas_organizations":                                                mongodbatlas.DataSourceOrganizations(),
+		"mongodbatlas_custom_dns_configuration_cluster_aws":                         customdnsconfigurationclusteraws.DataSource(),
+		"mongodbatlas_online_archive":                                               onlinearchive.DataSource(),
+		"mongodbatlas_online_archives":                                              onlinearchive.PluralDataSource(),
+		"mongodbatlas_ldap_configuration":                                           ldapconfiguration.DataSource(),
+		"mongodbatlas_ldap_verify":                                                  ldapverify.DataSource(),
+		"mongodbatlas_search_index":                                                 searchindex.DataSource(),
+		"mongodbatlas_search_indexes":                                               searchindex.PluralDataSource(),
+		"mongodbatlas_data_lake_pipeline_run":                                       datalakepipeline.DataSourceRun(),
+		"mongodbatlas_data_lake_pipeline_runs":                                      datalakepipeline.PluralDataSourceRun(),
```

Comment:
should we have these in a separate package from `data_lake_pipeline`?

Reply:
i was thinking about it but at the end I decided to have them together because there are not resources associated to them

Reply:
I see, if it has some relation to the data_lake_pipeline resource then I agree it makes sense